internal/test/clusterinfo/openshift.go

```diff
@@ -18,19 +16,10 @@ type OpenShift struct {
 	Client *clientset.Clientset
 }
 
-// GetOpenShift creates client for the current OpenShift cluster. If Kubeconfig is provided, it is used to create client,
-// otherwise it uses in-cluster config.
-func GetOpenShift(kubeConfigPath string) (*OpenShift, error) {
-	log.Printf("kubeconfig source: %s", kubeConfigPath)
-	var rc *rest.Config
-	var err error
-
-	if kubeConfigPath == "" {
-		// InClusterConfig uses default service account or service account provided by the pod to obtain config.
-		rc, err = rest.InClusterConfig()
-	} else {
-		rc, err = clientcmd.BuildConfigFromFlags("", kubeConfigPath)
-	}
+// GetOpenShift creates client for the current OpenShift cluster. If KUBECONFIG env var is set, it is used to
```

Comment:
looks like doc needs to be updated since we are not checking for kubeconfig env var here.